### ISSUE TITLE
Publish to open-vsx

### ISSUE THREAD

**Issue by checkraisefold** [CRBroughton/htmx-attributes#5]
Publish this extension to the open-source extension registry at https://open-vsx.org/

**Comment by CRBroughton** [comment]
Hi there! I've never heard of open-vsx, but I can certainly look into publishing the extension there soon. I'll update this issue if it's viable and once it's done. Thanks for opening up the issue!

**Comment by checkraisefold** [comment]
> Hi there! I've never heard of open-vsx, but I can certainly look into publishing the extension there soon. I'll update this issue if it's viable and once it's done. Thanks for opening up the issue!

VSCodium, the open-license distribution of VS Code uses open-vsx as its default registry; publishing it would allow VSCodium users to have your extension be installable without having to manually download the vsix as well.

**Comment by CRBroughton** [comment]
Yep it's all published now, so you should be good to download and install for VSCodium. I'll keep this open until the namespacing issue is resolved :)
